internal/pkg/v2/handler/core/data/domain_event.go

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package data
+
+// An event indicating that a given device has just reported some data
+type DeviceLastReported struct {
+	DeviceName string
+}
+
+// An event indicating that the service associated with the device that just reported data is alive.
+type DeviceServiceLastReported struct {
+	DeviceName string
+}
```

Comment:
Copy from v1 `internal/core/data/domain_event.go` for event.go(event handler).

Reply:
We won't control DeviceLastReporated and DeviceServiceLastReported in core-data anymore.  They will be handled by Device Service.  Please remove this file.

Reply:
Removed.